javalib/src/main/scala/java/util/Formatter.scala

```diff
@@ -114,6 +114,25 @@ final class Formatter(private val dest: Appendable) extends Closeable with Flush
             result
           }
 
+          def formatThousands(value: String, decimalSep: Char = '.', thousandsGrouping: String = ","): String = {
```

Comment:
Actually, just drop the last two arguments. They are not used as arguments anyway. This code is not user facing, so we are better off changing it when we need.